lesson_7/vlad_butulan/work2.py

```diff
@@ -0,0 +1,15 @@
+# Create a decorator called safe_divide that will output a message if the division cannot be performed, othervise it will return the result
+
+def safe_divide(fct):
+    def check(first_parameter, second_parameter):
+        if second_parameter == 0:
+            return "Division cannot be performed"
+        else:
+            return fct(first_parameter, second_parameter)
+    return check
+
```

Comment:
It is recommended that you add two spaces between functions

Reply:
fixed